src/stylesheets/base/_typography.scss

```diff
@@ -1,88 +1,110 @@
 // UNIVERSAL
+
 .hdr-text {
-  @include text( $primary-font, normal, 1.4rem, $color-dark-green );
+  @include text($primary-font, normal, 1rem, $color-dark-green);
   text-transform: uppercase;
-  letter-spacing: .2em;
-
+  letter-spacing: 0.2em;
+  text-align: center;
   @media (min-width: 550px) {
     font-size: 1.1rem;
   }
 }
 
 .intro-text {
-  @include text( $secondary-font, normal, 1.8rem, $color-grey );
+  @include text($secondary-font, normal, 1.8rem, $color-grey);
   text-align: center;
-
   @media (min-width: 550px) {
     font-size: 2.1rem;
     text-align: left;
   }
 }
 
 .context-text {
-  @include text( $primary-font, normal, 1.4rem, $color-grey );
+  @include text($primary-font, normal, 1.5rem, $color-grey);
   line-height: 2rem;
   text-align: center;
-  letter-spacing: .05em;
-
+  letter-spacing: 0.05em;
   @media (min-width: 550px) {
     text-align: left;
     letter-spacing: none;
   }
 }
-
 // NAVIGATION
+
 .nav__wrapper a {
-  @include text( $primary-font, normal, 1.1rem, $color-mint );
+  @include text($primary-font, normal, 1rem, $color-mint);
   text-transform: uppercase;
   text-decoration: none;
-
   @media (min-width: 550px) {
     font-size: 1.1rem;
   }
 }
-
 // PROBLEM SECTION
+
 .problem {
   &__sup {
-    @include text( $primary-font, normal, .7rem, $color-orange-red );
+    @include text($primary-font, normal, 0.7rem, $color-orange-red);
     text-decoration: underline;
 
     &:hover {
       font-weight: bold;
     }
-
     @media (min-width: 550px) {
       font-size: 1rem;
     }
   }
 
   &__quote {
-    @include text( $secondary-font, normal, 1.4rem, $color-brown );
+    @include text($secondary-font, normal, 1.4rem, $color-brown);
     text-align: left;
     line-height: 2.2rem;
-
     @media (min-width: 550px) {
       font-size: 1.4rem;
       text-align: center;
     }
   }
 
   &__author {
-    @include text( $primary-font, bold, 1.4rem, $color-grey );
+    @include text($primary-font, bold, 1.4rem, $color-grey);
     text-align: left;
     font-style: normal;
-
     @media (min-width: 550px) {
       font-size: 1.2rem;
       text-align: center;
     }
   }
 }
+// FACTORS SECTION
 
+.keyFactors {
```

Comment:
`key-factors` instead of `keyFactors`

Use `camelCase` in JavaScript. Use `kabob-case` in HTML and class names

Reply:
resolved in pr#8